dockerfiles/centos/streaming-integrator/Readme.md

```diff
@@ -0,0 +1,65 @@
+# Dockerfile for WSO2 Streaming Integrator #
+
+This section defines the step-by-step instructions to build an [CentOS](https://hub.docker.com/_/centos/)  Linux based Docker image
+for WSO2 Streaming Integrator 1.0.0.
+
+## Prerequisites
+
+* [Docker](https://www.docker.com/get-docker) v17.09.0 or above
+* [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) client
+
+## How to build an image and run
+
+##### 1. Checkout this repository into your local machine using the following Git client command.
+
+```
+git clone https://github.com/wso2/docker-ei.git
+```
+
+>The local copy of the `dockerfiles/centos/streaming-integrator` directory will be referred to as `SI_DOCKERFILE_HOME` from this point onwards.
+
+##### 2. Build the Docker image.
+
+- Navigate to `<SI_DOCKERFILE_HOME>` directory. <br>
+  Execute `docker build` command as shown below.
+    + `docker build -t wso2si:1.0.0-centos .`
+
+> By default, the Docker image will prepackage the General Availability (GA) release version of the relevant WSO2 product.
+
+##### 3. Running the Docker image.
+
+- `docker run -p 9443:9443 wso2si:1.0.0-centos`
+
+## How to update configurations
```

Comment:
@Aaquiff is this example tested with the new version `7.0.0`?

Reply:
@chirangaalwis this seems outdated. I will rewrite this with an example using the new configuration model.

Reply:
@chirangaalwis, this was taken from the Readme.md of the ubuntu implementation. Seems that this has been updated for 7.0.0. Should be good to go.